extension/public/static/manifest/v3.json

```diff
@@ -10,18 +10,12 @@
   },
   "background": {
     "service_worker": "background.min.js",
-    "scripts": [
-      "background.min.js"
-    ]
+    "scripts": ["background.min.js"]
```

Comment:
auto formatting from Husky because I committed this from my local machine. The pipeline does not run this cmd